pkgs/desktops/gnome/misc/gnome-extensions-cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
```

Comment:
Should not this use `buildPythonApplication`?

Reply:
I'm not sure, I can change it if this is preferred.

Reply:
@jtojnar I changed this in the latest commit. Looks like either would have worked, but buildPythonApplication is more appropriate. https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function

Can you review this PR, please?